models/articles-model.js

```diff
@@ -1,5 +1,19 @@
 const db = require("../db/connection");
 
+exports.checkArticleExists = (article_id) => {
```

Comment:
don't you already have a model in this file that promise rejects if the article doesn't exist? Why do you need to create another?